recipes/netcdf-cxx4/meta.yaml

```diff
@@ -0,0 +1,32 @@
+{% set version = "4.2.1" %}
+
+package:
+  name: netcdf-cxx4
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/Unidata/netcdf-cxx4
+  git_tag: v{{ version }}
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - libnetcdf
+  run:
+    - libnetcdf
```

Comment:
If I can help in any way, yes. I'll have closer look in a few days after finishing some other pending stuff.